src/compiler.ts

```diff
@@ -1651,6 +1651,7 @@ export class Compiler extends DiagnosticEmitter {
         break;
       }
       case NodeKind.FIELDDECLARATION: {
+        if (!this.program.elementsByDeclaration.has(<FieldDeclaration>statement)) { break; }
```

Comment:
nit: if expression and it's body contain in same line braces is unnecessary

Reply:
I assumed since it didn't fail a lint that it wasn't enforced either way, if there is a preference would it be reasonable to add the check into a lint rule on a new PR?